src/sans_fitter/inversion/__init__.py

```
@@ -0,0 +1,63 @@
+    'explore_dmax',
+    'PrResult',
+    'DmaxScan',
+    'AlphaEstimate',
+    'NTermsEstimate',
+    'InsufficientDataError',
+    'PrEstimationError',
+    'DEFAULT_N_TERMS',
+    'DEFAULT_R_POINTS',
+    'REGULARIZERS',
+    'SASVIEW_N_REG',
+]
```

Ruff (0.16.2)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)